examples/dlint/js.rs

```diff
@@ -103,14 +103,26 @@ fn op_query_control_flow_by_span(
   .map_err(Into::into)
 }
 
-pub struct JsRuleRunner {
+#[derive(Debug)]
+pub struct PluginRunner {
+  plugin_path: String,
+}
+
+impl PluginRunner {
+  pub fn new(plugin_path: &str) -> Box<Self> {
+    Box::new(Self {
+      plugin_path: plugin_path.to_string(),
+    })
+  }
+}
+
+struct JsRunner {
   runtime: JsRuntime,
   module_id: i32,
 }
```

Comment:
Previous `JsRuleRunner` struct can neither be `Send` nor `Sync` because of `JsRuntime`. So I split it into two, one is `PluginRunner`, which is thread safe, the other is `JsRunner`, which is not.
What we'll share across worker threads is `PluginRunner` only.